src/pyobject_converter.h

```diff
@@ -222,6 +223,7 @@ class PyObjectConverter : public Converter<PyObject *> {
         virtual PyObject * fromFloating(double v) { return PyFloat_FromDouble(v); }
         virtual PyObject * fromBoolean(bool v) { return PyBool_FromLong((long)v); }
         virtual PyObject * fromString(const char *v) { return PyUnicode_FromString(v); }
+        virtual PyObject * fromBytes(const char *v) { return PyBytes_FromString(v); }
```

Comment:
The `fromBytes()` function should also be added to `converter.h` as pure virtual function.